src/index.ts

```diff
@@ -1,7 +1,7 @@
 import path, { posix } from 'node:path';
 import { type Options as FdirOptions, fdir } from 'fdir';
 import picomatch from 'picomatch';
-import { escapePath, getPartialMatcher, isDynamicPattern, log, splitPattern } from './utils.ts';
+import { chalkLog, escapePath, getPartialMatcher, isDynamicPattern, log, splitPattern } from './utils.ts';
```

Comment:
it's a bit confusing to me to use the work chalk in the name when it's not using the `chalk` npm library. perhaps calling it `coloredLog` or something would make more sense

Reply:
Initially, I wasn't sure what variable name to use, so I used this name temporarily. 

Thanks for your suggestion, it seems reasonable. 👍